.github/workflows/cuda_build_test.yml

```diff
@@ -1,6 +1,6 @@
 name: CUDA Kernels Build
 
-on: [pull_request]
+on: []
 
 jobs:
   build-awkward-cuda-kernels:
```

Comment:
Actually, you can get rid of the cuda_build_test entirely. All it did before was check that the kernels compile; now we can't even do that because we'd need CuPy to be installed.

We'll need something else for testing, though I don't know what that can be. I've brought up the question with colleages.